backend/src/main/java/ai/giskard/service/InitService.java

```diff
@@ -143,19 +145,30 @@ public String getProjectByCreatorLogin(String login) {
         return projects.entrySet().stream().filter(e -> e.getValue().creator.equals(login)).findFirst().orElseThrow().getValue().name;
     }
 
+    static final GalleryUnlockIndicator lockIndicator = new GalleryUnlockIndicator();
+    public static boolean isUnlocked() { return lockIndicator.isUnlocked(); }
+    public static void setUnlocked(boolean unlocked) { lockIndicator.setUnlocked(unlocked); }
+
     /**
      * Initializing first authorities, mock users, and mock projects
      */
     @EventListener(ApplicationReadyEvent.class)
     public void init() {
         projects = createProjectConfigMap();
         generalSettingsService.saveIfNotExists(new GeneralSettings());
+        // change readonly
         initAuthorities();
         initUsers();
+        initDefaultApiKey();
         List<String> profiles = Arrays.asList(env.getActiveProfiles());
         if (!profiles.contains("prod") && !profiles.contains("dev")) {
             initProjects();
         }
+        lockIndicator.setUnlocked(false);
+    }
+
+    private void initDefaultApiKey() {
+        apiKeyService.initDefaultApiKey();
```

Comment:
Why don't we just call the service line 162?

Reply:
Thought about some more operations for it but finally dropped.
Will improve it